### ISSUE TITLE
Bug: Cache requires reset after update

### ISSUE THREAD

**Issue by labrys** [SickRage/sickrage-issues#126]
After an update to the dev branch, SR may fail to start with a Mako error.  This is due to the changes in the gui.  Deleting `cache/mako` and `cache/sessions` resolves this error.  We should not merge dev to master until we resolve this conflict.  Potentially add a purge cache on load or something similar.

**Comment by labrys** [comment]
@SickRage/collaborators @SickRage/moderators

**Comment by MGaetan89** [comment]
Just testing my memory here, so I might be wrong.
I think the `cache/mako` folder is cleared when updating SickRage.
However, it is possible that it is not cleared when changing branches.

**Comment by duramato** [comment]
@labrys iir it already has something to clear mako cache , and i never got that errors on latest dev

**Comment by fernandog** [comment]
@Labrys please follow guidelines lol

**Comment by miigotu** [comment]
The error happens when the backend changes because the python is not completely reloaded when restarting after an update.

**Comment by OmgImAlexis** [comment]
@miigotu what if we just make SickRage restart after an update?

**Comment by miigotu** [comment]
we do, its just that the current one is still running when the first one is starting. No real way around that using popen

**Comment by OmgImAlexis** [comment]
Mark as "won't fix" then?

**Comment by miigotu** [comment]
yep, its only when we change what vars are passed to templates in webserve anyways
